setup.py

```diff
@@ -12,7 +12,7 @@
     'python-dateutil==2.7.5',
     'boto3',
     'slackclient>=2.0.0',
-    'pandas',
+    'pandas<0.25.0',
```

Comment:
I think it's better to lock pandas version because there is a difference in pickle loading due to version differences.
For example, pandas 0.24.x can't load pickle made with pandas 0.23.x.
https://stackoverflow.com/questions/54665527/importerror-no-module-named-pandas-core-internals-managers-pandas-core-inte